docs/channel.rst

```diff
@@ -460,72 +460,4 @@ You can specify more than one of these events by using a comma separated string
 See also: `fromPath`_ factory method.
 
 
-Channel methods
-===============
-
-.. _channel-subscribe:
-
```

Comment:
Please add this line to the new location:
```
.. _operator-subscribe:
```

I didn't find any links to it in the docs though

Reply:
The section you asked me to add `subscribe` now links to it through this :)